judgels-backends/judgels-commons/judgels-persistence-core/src/main/java/judgels/persistence/hibernate/UnmodifiableHibernateDao.java

```diff
@@ -49,7 +51,12 @@ public M insert(M model) {
         model.createdAt = clock.instant();
         model.createdIp = actorProvider.getIpAddress().orElse(null);
 
-        return persist(model);
+        return super.persist(model);
+    }
+
+    @Override
+    public M persist(M model) {
```

Comment:
Why override if it is just calling super

Reply:
Fixed